.projenrc.js

```diff
@@ -105,3 +115,74 @@ project.eslint.addRules({
 });
 
 project.synth();
+
+function codeGenFetchAssemblies() {
```

Comment:
What's the value of generating this code instead of just writing it directly? Not sure I completely get it

Reply:
Yeah, it started of way more dynamic than what it is now - ejected.